context/app/static/js/components/home/FacetSearchEntityListItems/FacetSearchEntityListItems.jsx

```diff
@@ -11,18 +11,14 @@ function FacetSearchEntityListItems({ entityType, matches, labels }) {
             <FacetLabel color="primary" variant="subtitle2">{`${labels[k]} (${entityType}s)`}</FacetLabel>
             <StyledList>
               {v.map((m) => {
-                const facetQueryParam =
-                  k === 'mapped_data_access_level'
-                    ? `mapped_status-mapped_data_access_level[0][0]=Published&mapped_status-mapped_data_access_level[1][0]=${encodeURIComponent(
-                        m.key,
-                      )}`
-                    : `${encodeURIComponent(k)}[0]=${encodeURIComponent(m.key)}`;
                 return (
                   <li key={m.key}>
                     <FacetValue
                       variant="body2"
                       component="a"
-                      href={`/search?entity_type[0]=${entityType}&${facetQueryParam}`}
+                      href={`/search?entity_type[0]=${entityType}&${encodeURIComponent(k)}[0]=${encodeURIComponent(
+                        m.key,
+                      )}`}
```

Comment:
```suggestion
                      href={`/search?entity_type[0]=${entityType}&${facetQueryParam}`}
```
Slight preference to keep the variable, for readability.